frontend/screens/BrandRegistrationForm.js

```diff
@@ -267,6 +270,13 @@ const styles = StyleSheet.create({
     flexDirection: "row",
     width: 358,
   },
+  loginFrame: {
```

Comment:
Lets create a `signup` folder and all all the signup screens in there along with a signup.scss file where the all the styles related to signup can reside